src/plugins/expressions/common/execution/execution.ts

```diff
@@ -415,4 +416,28 @@ export class Execution<
     // function which would be treated as a promise
     return { resolvedArgs };
   }
+
+  public async interpret<T>(
+    ast: ExpressionAstNode,
+    input: T,
+    options?: ExpressionExecOptions
+  ): Promise<unknown> {
+    switch (getType(ast)) {
+      case 'expression':
+        const execution = this.params.executor.createExecution(
+          ast as ExpressionAstExpression,
+          undefined,
```

Comment:
don't pass undefined, add context to the parameters

Reply:
Done.